### ISSUE TITLE
Unable to expand withdrawn controls

### ISSUE THREAD

**Issue by kylelaker** [EasyDynamics/oscal-react-library#743]
### Description

A control can itself have controls within it (for example, SP 800-53 uses this to represent control enhancements). A control may also be withdrawn (represented by a prop where `@name="status"` and `@value="withdrawn" in the appropriate namespace).

A user may want to be able to view the details (prose and enhancements) of a withdrawn control; however, in the application today, the ability to expand a control is disabled when a control is withdrawn. This makes viewing these details impossible. In SP 800-53rev5, these details seem to often not be present; however, other catalogs may choose to still provide prose and other information for withdrawn controls.

### Environment

oscal-viewer.msd.easydynamics.com

### Steps to Reproduce

*A catalog to reproduce will be provided soon*

### Expected behavior

I should be able to expand the detail view of a withdrawn control if there's something to see there.

### Actual Behavior

It is never possible to expand a withdrawn control.

### Additional Notes

We get input from @brian-easyd to find out what it means if a "parent" control is withdrawn but the inner controls are not labeled as withdrawn. For example:

```json
{
  "controls": [
    {
      "id": "repro-A",
      "title": "Sample Withdrawn",
      "props": [
        { "name": "status", "value": "withdrawn" }
      ],
      "controls": [ {"id": "repro-A.1", "title": "Parent is withdrawn but this isn't labeled as such" } ]
    }
  ]
}
```

Should `repro-A.1` have strikethrough? Or should it appear normally?

**Comment by brian-easyd** [comment]
Ensure consistent styling of text for withdrawn content.

If a parent control is withdrawn, all child controls are considered withdrawn as well.

**Comment by brian-easyd** [comment]
Display of withdrawn controls should include the same content as a normal control. It should be formatted differently and display some kind of "Withdrawn" label.

**Comment by kylelaker** [comment]
Closed by #720
